### PR TITLE
Add landuse=religious preset

### PR DESCRIPTION
**Pull request by YuliyaVeklicheva**
for #4476

### CODE REVIEW

File: data/presets/presets/landuse/religious.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "geometry": [
+    "area"
+  ],
+  "tags": {
```
Comment (bhousel) [inline data/presets/presets/landuse/religious.json]: This is great thank you!
Can you also add a name field into this file too?
```
    "fields": [
        "name"
    ],
```

Comment (bhousel) [reply data/presets/presets/landuse/religious.json]: (and then run `npm run build` again)

**Comment by YuliyaVeklicheva** [comment]
@bhousel updated

**Comment by bhousel** [comment]
Great thank you @YuliyaVeklicheva !
The name shows up strangely because there is no icon, but that is a different bug with the label placement code that I will ticket separately.

**Pull request merged**